src/app/_components/header-items.tsx

```diff
@@ -38,35 +64,30 @@ const MobileNav = (props: { isLoggedIn: boolean }) => {
         </div>
 
         <Collapsible.Content>
-          <div className="mx-4 my-2 flex flex-col gap-2">
-            <Link
-              className="text-right"
-              target="_blank"
-              href={RoutePath.WEBSITE_URL}
-            >
-              Visit Website
-            </Link>
-            <Link
-              className="text-right"
-              target="_blank"
-              href={RoutePath.WEBSITE_FAQ_URL}
-            >
-              Visit FAQ
-            </Link>
-            {isLoggedIn && (
-              <Link className="text-right" href={RoutePath.LOGOUT_PAGE}>
-                Logout
-              </Link>
-            )}
+          <div className="mx-4 my-2 flex flex-col gap-6">
+            {navOptions.map((option) => {
+              const { label, href, target } = option;
+              return (
+                <Link
+                  key={label}
+                  className="text-right text-xl"
+                  target={target}
+                  href={href}
+                  onClick={() => setIsOpen(false)}
```

Comment:
Test this?

Reply:
Tested

https://github.com/barkbank/barkbank-pawtal/pull/111/commits/d7ed6c20ec685c658d8bb80eb14d2d3f63a75630